scripts/check-release-please.mjs

```diff
@@ -0,0 +1,82 @@
+import { strict as assert } from 'assert';
+import * as childProcess from 'child_process';
+import path from 'path';
+import { readFileSync } from 'fs';
+
+/*
+	Enumerates through
+
+	- lerna packages,
+	- release please's manifest, and
+	- release please's config, and
+
+	makes sure all those lists are in sync. If not, guides you to do the correct thing
+*/
+
+const errors = [];
+const logErrorIf = (condition, errorMessage) => {
```

Comment:
```suggestion
function logErrorIf(condition, errorMessage) {
```
or maybe `recordErrorIf`

There are other places where functions are written as `const foo = () => {`. Is there a reason to prefer this over the function keyword?

Reply:
I tend to use arrow-functions whenever I don't need `this`. Especially with pure functions. I wouldn't do this change.